### PR TITLE
mpv: 0.32.0 -> 0.33.0

### PR DESCRIPTION
**Pull request by Ma27**
###### Motivation for this change

https://github.com/mpv-player/mpv/releases/tag/v0.33.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by RaghavSood** [comment]
Result of `nixpkgs-review pr 104716` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>mpc-qt</li>
  </ul>
</details>
<details>
  <summary>17 packages built:</summary>
  <ul>
    <li>celluloid</li>
    <li>curseradio</li>
    <li>jellyfin-mpv-shim</li>
    <li>jftui</li>
    <li>minitube</li>
    <li>mpv</li>
    <li>mpv-unwrapped</li>
    <li>mpvScripts.autoload</li>
    <li>mpvScripts.mpris</li>
    <li>plex-media-player</li>
    <li>plex-mpv-shim</li>
    <li>python37Packages.mpv</li>
    <li>python38Packages.mpv</li>
    <li>qimgv</li>
    <li>somafm-cli</li>
    <li>sublime-music</li>
    <li>tuijam</li>
  </ul>
</details>

File: pkgs/applications/video/mpv/default.nix [hunk 1]
```diff
@@ -39,13 +39,8 @@
 , libpngSupport      ? true,           libpng        ? null
 , pulseSupport       ? config.pulseaudio or stdenv.isLinux, libpulseaudio ? null
 , rubberbandSupport  ? stdenv.isLinux, rubberband    ? null
-# NOTE: samba support should be removed on the next mpv release, see also:
-# https://github.com/NixOS/nixpkgs/pull/89145#issuecomment-636424362
-# Please remove this line on the next mpv release.
-, sambaSupport       ? false,          samba         ? null
 , screenSaverSupport ? true,           libXScrnSaver ? null
 , sdl2Support        ? true,           SDL2          ? null
-, sndioSupport       ? true,           sndio         ? null
```
Comment (AndersonTorres) [inline pkgs/applications/video/mpv/default.nix:48]: Was this removed upstream?

**Comment by AndersonTorres** [comment]
I tested mpc-qt locally. No problems here.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
LGTM, LGTBorg.

**Review by AndersonTorres — APPROVED** [review]

**Pull request merged**